sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -1800,8 +1800,23 @@ BRACKETED_EMPTY_COMMENT
     : '/**/' -> channel(HIDDEN)
     ;
 
+// The bracketed comment consists of six main parts, from left to right are:
+// "/*", ~[+], left part, nested bracketed comment, right part, "*/".
+//   Ⅰ. '/*' is used to match the beginning of a bracketed comment.
```

Comment:
it's odd to use roman numerals, can we just use 1, 2, 3?

Reply:
OK